### DISCUSSION TITLE
Is the site revanced.io official and/or safe?

### DISCUSSION THREAD

**Discussion by arandomhooman** [ReVanced/revanced-discussions#740] [Question] [answered]
Is it?

**Comment by ghost** [comment, marked as answer]
No, revanced.app is the only official website

**Reply by arandomhooman** [reply]
Could the site be mal?

**Reply by ghost** [reply]
Could be.

**Reply by arandomhooman** [reply]
They claim ownership to revanced services and visuals.

**Reply by ghost** [reply]
ReVanced is a community effort. Only official website for ReVanced is revanced.app
